new-dti-website/components/team/TeamAbout.tsx

```diff
@@ -121,59 +121,72 @@ const TeamStatistics = () => {
       return acc;
     }, new Set());
 
+  const countColleges = (role?: string): College[] =>
+    allMembers
+      .filter((member) => !role || role === (getGeneralRole(member.role) as string))
+      .reduce(
+        (acc, val) => (val.college === undefined ? acc : [...acc, val.college]),
+        [] as College[]
+      );
+
   const emptyRoleStats: roleStatistics = {
     designer: {
       name: 'Design',
       color: '#FFBCBC',
       people: 0,
       majors: new Set(),
-      colleges: new Set()
+      colleges: []
     },
     developer: {
       name: 'Development',
       color: '#D63D3D',
       people: 0,
       majors: new Set(),
-      colleges: new Set()
+      colleges: []
     },
-    pm: { name: 'Product', color: '#FFFFFF', people: 0, majors: new Set(), colleges: new Set() },
+    pm: { name: 'Product', color: '#FFFFFF', people: 0, majors: new Set(), colleges: [] },
     business: {
       name: 'Business',
       color: '#B7B7B7',
       people: 0,
       majors: new Set(),
-      colleges: new Set()
+      colleges: []
     },
-    lead: { name: 'Leads', color: '#484848', people: 0, majors: new Set(), colleges: new Set() }
+    lead: { name: 'Leads', color: '#484848', people: 0, majors: new Set(), colleges: [] }
   };
 
   const roleStats = populateObject(emptyRoleStats, (key, value) => {
     const count = allMembers.filter((mem) => key === (getGeneralRole(mem.role) as string)).length;
-    return { ...value, people: count, majors: countMajors(key) as Set<string> };
+    return {
+      ...value,
+      people: count,
+      majors: countMajors(key) as Set<string>,
+      colleges: countColleges(key)
```

Comment:
Just curious, we are we using a list over a set? I think that this logic doesn't handle duplicates, does it? If you want I just populated some college data on IDOL and pulled it into the JSON. feel free to rebase to double check.

Reply:
There are only so many colleges, so I didn't think it was really worth to use a set. Good catch, I forgot to add the duplicate handling logic.